### ISSUE TITLE
remove EnvUpdate from v2 docs

### ISSUE THREAD

**Issue by mmikeww** [Lexikos/AutoHotkey_L-Docs#163]
v2changes says that this command is removed, but it still shows up in the command list index:

https://lexikos.github.io/v2/docs/commands/index.htm

**Comment by Lexikos** [comment]
That was intentional.  It's still in the index because it still has its own page.

I removed the command because:

  - It's not really connected to EnvGet/EnvSet as the name implies, since it doesn't use the script's own env vars and doesn't affect the script itself in any way.
  - It can easily be replaced with a single line (SendMessage).
  - I think it's rarely used.

But then I added it to stdlib to help anyone that actually uses it, so left it in the documentation.  At some point I stopped including the stdlib files in the download.  I don't think there's really any benefit in moving it to the stdlib, so now I'm not sure whether to remove it completely or just put it back in and forget about it.

I did some testing on Windows 10 and found that EnvUpdate has even more limited use than I thought.  A command prompt opened from Explorer shows the updated values, but somehow other programs launched by Explorer (such as AutoHotkey) do not, unless Explorer is also restarted.  If the command/documentation is kept, I suppose that its limitations should be mentioned.
